### PR TITLE
Feature/redux

### PR DESCRIPTION
**Pull request by chrissinclairedited**

### CODE REVIEW

File: frontend/src/data/shards/actions.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    LOAD_SHARD_STATUS_SUCCESS: 'LOAD_SHARD_STATUS_SUCCESS',
+    LOAD_SHARD_STATUS_ERROR: 'LOAD_SHARD_STATUS_ERROR',
+};
+
+export function loadShardStatus() {
+    return async (dispatch) => {
+        dispatch({
+			type: SHARD_ACTION_TYPES.LOAD_SHARD_STATUS,
+		});
+
```
Comment (gchazot) [inline frontend/src/data/shards/actions.js]: Looks like there is some indentation issue here (1 too many + maybe indented with tabs?)

File: frontend/src/data/shards/actions.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+				throw new Error(`Bad response from server: ${response.statusText}`);
+			}
+		} catch (error) {
+			dispatch({
+				type: SHARD_ACTION_TYPES.LOAD_SHARD_STATUS_ERROR,
+				error,
+			});
+		}
+	}
+}
```
Comment (gchazot) [inline frontend/src/data/shards/actions.js]: missing semicolon?
Maybe worth adding prettier to the project...

File: frontend/src/ShardsDashboard.jsx [hunk 3]
```diff
@@ -1,15 +1,64 @@
...
-        return (
-            <div>
-                Hello from shards!
-            </div>
-        )
-    }
-}
+		const { loadingState } = this.props;
+
+		if (loadingState === 'NOT_LOADED' || loadingState === 'LOADING') {
```
Comment (gdelavald) [inline frontend/src/ShardsDashboard.jsx]: ooooh we can use suspense here (not needed in this PR though)

File: frontend/src/data/shards/actions.js (hunk 2 above)
Comment (chrissinclairedited) [reply frontend/src/data/shards/actions.js]: I don't think we generally have semi-colons after function definitions. Agree on prettier though - no point in having stupid stuff like this slow us down.

File: frontend/src/data/shards/actions.js (hunk 1 above)
Comment (chrissinclairedited) [reply frontend/src/data/shards/actions.js]: Yikes for sure. Will have a quick swizz at what's happened there.

File: frontend/src/ShardsDashboard.jsx (hunk 3 above)
Comment (chrissinclairedited) [reply frontend/src/ShardsDashboard.jsx]: Yeah totally could! Probably for a separate PR though as you say.

**Comment by chrissinclairedited** [comment]
Thanks for the reviews @gchazot @gdelavald!

**Pull request merged**